.travis.yml

```diff
@@ -1,6 +1,19 @@
 language: rust
 rust:
     - nightly
+
+before_script:
+    - "sh -e /etc/init.d/xvfb start"
+    - "export DISPLAY=:99.0"
+    - "wget http://selenium-release.storage.googleapis.com/2.50/selenium-server-standalone-2.50.1.jar"
```

Comment:
I found a [npm package](https://www.npmjs.com/package/selenium-standalone) which puts some sugar to installation:
```sh
npm install selenium-standalone@latest -g
selenium-standalone install
selenium-standalone start
```